husky_bringup/launch/robotiq_s_model_grippers/robotiq_s_gripper_left.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
```

Comment:
Parametrizing WRT left and right won't work - these scripts are launched via /etc/ros/indigo/husky-core.d where parameters cannot be passed to the launch files without modification to the existing system. 

Henceforth, only a merge can be done by hard coding the two situations into one launch file.